example/retry_server.js

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
We can remove this file as the retry mechanism is already incorporated in `servers/mockserver.js`